### ISSUE TITLE
[Bug]: [People picker][a11y][Voice over][Mobile] People picker not able to move cursor position to `Start/End` points between characters with double tap

### ISSUE THREAD

**Issue by sahilkumarmaths** [microsoft/fluentui#28007]
### Library

React / v8 (@fluentui/react)

### System Info

```shell
Test Environment:
Device: iPhone XR
iOS Version: 16.4.1 (a)
Screen Reader: Voiceover
```

### Are you reporting Accessibility issue?

None

### Reproduction

https://developer.microsoft.com/en-us/fluentui#/controls/web/peoplepicker

## Actual Behavior
Users are not able to move cursor position at 'Start/End' points, 'Between Characters' when user double taps, swipe up or down on text field with voiceover gestures in the people picker.

## Expected Behavior
Users should be able to move cursor position at 'Start/End' points, 'Between Characters' when user double taps, swipe up or down on text field with voiceover gestures in the people picker.

### Requested priority

Blocking

### Are you willing to submit a PR to fix?

yes

### Validations

- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] The provided reproduction is a minimal reproducible example of the bug.

**Comment by sahilkumarmaths** [comment]
@Hotell @smhigley : Can you please confirm the ETA for this? Thank you.

**Comment by smhigley** [comment]
This is because it is a combobox, not a plain input. You can see the same behavior on the ARIA Practices combobox: https://www.w3.org/WAI/ARIA/apg/patterns/combobox/examples/combobox-autocomplete-both/

Generally web authors have no control over AT-specific interactions, beyond defining the correct semantics. In this case, we have a native text input with a `role="combobox"`. VoiceOver-specific interactions are in the control of VoiceOver, not the web author.
